README.md

```diff
@@ -235,23 +235,37 @@ Batching is particularly useful with GraphQL. Using such techniques as preloadin
 Let's take a look at the simple [graphql-ruby](https://github.com/rmosolgo/graphql-ruby) schema example:
 
 ```ruby
-Schema = GraphQL::Schema.define do
-  query QueryType
+class MyprojectSchema < GraphQL::Schema
+  mutation(Types::MutationType)
+  query(Types::QueryType)
+
+  use BatchLoader::GraphQL
 end
 
-QueryType = GraphQL::ObjectType.define do
-  name "Query"
-  field :posts, !types[PostType], resolve: ->(obj, args, ctx) { Post.all }
+module Types
+  class QueryType < Types::BaseObject
+    field :posts, ![PostType]
```

Comment:
could we use [null: false](https://graphql-ruby.org/type_definitions/objects) vs `!`?


here is an example from the tests:

https://github.com/exAspArk/batch-loader/blob/9b015fb3f57c8783d7c3e780ed5596b46e1dbea2/spec/fixtures/graphql_schema.rb#L33-L66